### PR TITLE
Updated UI with new design 

### PR DESCRIPTION
**Pull request by preetham-coupa**
## Code reviewers

- [ ] @preetham-p-m 

## Summary of issue
Updated UI with new view page

## Summary of change
ui changes

## Checklist - reminder for author and reviewers

- [ ] WCAG
- [ ] Security considerations (ACL)
- [ ] Deep-links
- [ ] Clean and readable url
- [ ] Performance considerations
- [ ] Clean code
- [ ] Unit tests are provided/updated

### CODE REVIEW

File: client-app/src/components/Activity/Dashboard/Filters/ActivityFilters.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <>
+      <Menu vertical size="large" style={{ width: '100%', marginTop: 25 }}>
+        <Header icon="filter" attached color="teal" content="Filters" />
+        <Menu.Item content="All Activities" />
+        <Menu.Item content="I'm going" />
+        <Menu.Item content="I'm hosting" />
+      </Menu>
+      <Header />
+      <Calendar />
```
Comment (preetham-p-m) [inline client-app/src/components/Activity/Dashboard/Filters/ActivityFilters.tsx:14]: format this file

File: client-app/src/components/Activity/Details/ActivityDetailedHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+};
+
+const activityImageTextStyle = {
+  position: 'absolute',
+  bottom: '5%',
+  left: '5%',
+  width: '100%',
+  height: 'auto',
+  color: 'white'
+};
```
Comment (preetham-p-m) [inline client-app/src/components/Activity/Details/ActivityDetailedHeader.tsx:20]: why we have styling here

File: client-app/src/styles.css [hunk 3]
```diff
@@ -43,3 +43,7 @@ body {
   font-size: 1.7em;
   font-weight: normal;
 }
+
+/* .react-datepicker-wrapper {
+  width: 100%;
+} */
```
Comment (preetham-p-m) [inline client-app/src/styles.css:49]: why it is commented

Comment (preetham-coupa) [reply client-app/src/styles.css:49]: in from if date picker does not expand to 100%, we can use this, just for safety

File: client-app/src/components/Activity/Details/ActivityDetailedHeader.tsx (hunk 2 above)
Comment (preetham-coupa) [reply client-app/src/components/Activity/Details/ActivityDetailedHeader.tsx:20]: for background image

File: client-app/src/components/Activity/Dashboard/Filters/ActivityFilters.tsx (hunk 1 above)
Comment (preetham-coupa) [reply client-app/src/components/Activity/Dashboard/Filters/ActivityFilters.tsx:13]: done

**Review by preetham-p-m — APPROVED** [review]

**Pull request merged**